### PR TITLE
Add guide for Writefreely

### PR DESCRIPTION
**Pull request by thisven**
Please save the [icon](https://writefreely.org/img/icon.svg) to _static/images/writefreely.svg to serve it in this guide since I don't have write access to this location.

### CODE REVIEW

**Comment by luto** [comment]
Thanks for the guide! I will take a look later. Just one quick note, you can upload the icon yourself in your fork here: https://github.com/thisven/lab/tree/patch-1/source/_static/images

File: source/guide_writefreely.rst [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+  [isabell@stardust mailtrain]$ mysql -e "CREATE DATABASE ${USER}_writefreely CHARACTER SET latin1 COLLATE latin1_swedish_ci;"
+  [isabell@stardust mailtrain]$
+
+Writefreely Config
+------------------
+
+Change to _writefreely_ directory and start config mode to create a config file:
+
+.. note:: You need to select ``Production, behind reverse proxy`` in Server setup.
```
Comment (luto) [inline source/guide_writefreely.rst:75]: Please move this information out of the note box and after the command block. Also add the follow information needed to complete the wizard:

* confirm default listening port
* select MySQL as the database
* enter mysql credentials shown earlier
* use `<username>_writefreely`, replacing `<username>` with your username
* confirm MySQL host and port
* enter admin and blog details as needed
* enter your domain with `https://` and without port, e.g. `https://isabell.uber.space`
* choose other options as you see fit

File: source/guide_writefreely.rst [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+  
+Alter _bind_ key and add TLS certificates paths in ``config.ini``; look for the ``[server]`` block:
+
+.. code-block:: ini
+
+  ...
+
+  bind                 = 0.0.0.0
+  tls_cert_path        = /home/isabell/etc/certificates/isabell.uber.space.crt
+  tls_key_path         = /home/isabell/etc/certificates/isabell.uber.space.key
```
Comment (luto) [inline source/guide_writefreely.rst:91]: According to [the docs](https://writefreely.org/docs/latest/admin/config) a certificate is only needed when serving HTTPS directly using writefreely. On an uberspace HTTPS is handled by our nginx frontend, so there is no need to pass the certs here.

File: source/guide_writefreely.rst [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+Learn about other configuration options in the documentation's `admin guide`_.
+
+Key generation
+--------------
+
+Generate the encryption keys for your instance:
+
+.. code-block:: console
+
+  [isabell@stardust writefreely]$ writefreely keys generate
```
Comment (luto) [inline source/guide_writefreely.rst:104]: `writefreely` is not in the `$PATH`, so this needs to be `./writefreely`.
```suggestion
  [isabell@stardust writefreely]$ ./writefreely keys generate
```

File: source/guide_writefreely.rst [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+Supervisord Daemon Setup
+------------------------
+
+Create ``~/etc/services.d/writefreely.ini`` with the following content:
+
+.. code-block:: ini
+
+  [program:writefreely]
+  directory=%(ENV_HOME)s/writefreely/
+  command=%(ENV_HOME)s/writefreely/writefreely -c %(ENV_HOME)s/writefreely/config.ini
```
Comment (luto) [inline source/guide_writefreely.rst:123]: writefreely defaults to `config.ini`, so there is no need to pass it here.
```suggestion
  command=%(ENV_HOME)s/writefreely/writefreely
```

File: source/guide_writefreely.rst [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+.. code-block:: console
+
+  [isabell@stardust mailtrain]$ mysql -e "CREATE DATABASE ${USER}_writefreely CHARACTER SET latin1 COLLATE latin1_swedish_ci;"
+  [isabell@stardust mailtrain]$
+
+Writefreely Config
+------------------
+
+Change to _writefreely_ directory and start config mode to create a config file:
```
Comment (luto) [inline source/guide_writefreely.rst:73]: The `_something_` syntax using underscores does not do anything in restructured text, so the `_`s are [shown in the rendered output](https://deploy-preview-813--uberlab.netlify.app/guide_writefreely.html#writefreely-config). Please remove them throughout the guide or change them to something else.

File: source/guide_writefreely.rst [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+
+Database Setup
+--------------
+
+Create a new database:
+
+.. code-block:: console
+
+  [isabell@stardust mailtrain]$ mysql -e "CREATE DATABASE ${USER}_writefreely CHARACTER SET latin1 COLLATE latin1_swedish_ci;"
+  [isabell@stardust mailtrain]$
```
Comment (luto) [inline source/guide_writefreely.rst:68]: We're not in the mailtrain guide :grin: 
```suggestion
  [isabell@stardust ~]$ mysql -e "CREATE DATABASE ${USER}_writefreely CHARACTER SET latin1 COLLATE latin1_swedish_ci;"
  [isabell@stardust ~]$
```

File: source/guide_writefreely.rst [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+Writefreely Config
+------------------
+
+Change to _writefreely_ directory and start config mode to create a config file:
+
+.. note:: You need to select ``Production, behind reverse proxy`` in Server setup.
+
+.. code-block:: console
+
+  [isabell@stardust writefreely]$ cd ~/writefreely
```
Comment (luto) [inline source/guide_writefreely.rst:79]: The prompt changes only after `cd`. Before that we're in the default directory home / `~`.
```suggestion
  [isabell@stardust ~]$ cd ~/writefreely
```

**Review by luto — CHANGES_REQUESTED** [review]
Thanks for the new guide! I left a couple of comments to make the process more streamlined for people using it.

File: source/guide_writefreely.rst (hunk 2 above)
Comment (thisven) [reply source/guide_writefreely.rst:91]: Ok, i see.

File: source/guide_writefreely.rst (hunk 1 above)
Comment (thisven) [reply source/guide_writefreely.rst]: Done!

**Pull request merged**